hack/vendor.sh

```diff
@@ -1,28 +1,37 @@
 #!/usr/bin/env bash
 
-# This file is just wrapper around vndr (github.com/LK4D4/vndr) tool.
-# For updating dependencies you should change `vendor.conf` file in root of the
-# project. Please refer to https://github.com/LK4D4/vndr/blob/master/README.md for
-# vndr usage.
+# This file is just wrapper around 'go mod vendor' tool.
+# For updating dependencies you should change `vendor.mod` file in root of the
+# project.
 
 set -e
 
-if ! hash vndr; then
-	echo "Please install vndr with \"go get github.com/LK4D4/vndr\" and put it in your \$GOPATH"
-	exit 1
-fi
+cat > go.mod << EOF
+module github.com/moby/moby
+
+go 1.17
+EOF
+
+cat > hack/make/.resources-windows/go.mod << EOF
+module github.com/docker/docker/autogen/winresources/dockerd
+
+go 1.17
+EOF
 
 if [ $# -eq 0 ] || [ "$1" = "archive/tar" ]; then
```

Comment:
@thaJeztah I think this should do the trick!

Reply:
Actually, I've misunderstood this initially, looks like `internal/archive/tar` is not really needed, since download & patching is done each time this script runs... I was assuming it's options (clearly didn't read this script fully).